### PR TITLE
test: add tests for GenerateCloudFormationTemplate

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/deploy/rdws_test.go [hunk 1]
```diff
@@ -18,6 +22,36 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestRdwsDeployer_GenerateCloudFormationTemplate(t *testing.T) {
+	t.Run("ensure resulting CloudFormation template custom resource paths are empty", func(t *testing.T) {
+		// GIVEN
+		backend := mockBackendServiceDeployer()
```
Comment (iamhopaul123) [inline internal/pkg/cli/deploy/rdws_test.go]: 🤔

**Review by iamhopaul123 — COMMENTED** [review]
lgtm overall just one nit 🚢

File: internal/pkg/cli/deploy/rdws_test.go (hunk 1 above)
Comment (efekarakus) [reply internal/pkg/cli/deploy/rdws_test.go]: lol i noticed that too, just fixed it!

File: internal/pkg/cli/deploy/backend_test.go [hunk 2]
```diff
@@ -16,6 +20,40 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestBackendSvcDeployer_GenerateCloudFormationTemplate(t *testing.T) {
+	t.Run("ensure resulting CloudFormation template custom resource paths are empty", func(t *testing.T) {
```
Comment (Lou1415926) [inline internal/pkg/cli/deploy/backend_test.go:24]: qq: what about paths like addons path? do we want to keep  them empty as well ?

**Review by Lou1415926 — APPROVED** [review]

File: internal/pkg/cli/deploy/backend_test.go (hunk 2 above)
Comment (efekarakus) [reply internal/pkg/cli/deploy/backend_test.go:24]: ooh good thinking, let me test that with overrides and see if CDK complains. If it does i'll fix that too

**Review by iamhopaul123 — APPROVED** [review]

**Pull request merged**